anta/tests/snmp.py

```diff
@@ -237,3 +249,73 @@ def test(self) -> None:
             self.result.is_failure(f"Expected `{self.inputs.contact}` as the contact, but found `{contact}` instead.")
         else:
             self.result.is_success()
+
+
+class VerifySNMPErrors(AntaTest):
+    """Verifies the number of SNMP error counter(s) processed.
+
+    By default, all  error counters will be checked for any non-zero values.
+    An optional list of specific error counters can be provided for granular testing.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the SNMP error counter(s) are zero/None.
+    * Failure: The test will fail if the SNMP error counter(s) are non-zero/not None/Not Found or is not configured.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.snmp:
+      - VerifySNMPErrors:
+          error_counters:
+            - inVersionErrs
+            - inBadCommunityNames
+    """
+
+    name = "VerifySNMPErrors"
+    description = "Verifies the number of SNMP error counter(s) processed."
+    categories: ClassVar[list[str]] = ["snmp"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show snmp", revision=1)]
+
+    class Input(AntaTest.Input):
+        """Input model for the VerifySNMPErrors test."""
+
+        error_counters: (
+            list[
+                Literal[
```

Comment:
since we already defined the error list then use the same here so me should not update it on multiple places

Reply:
Updated. Added model validator.  Thanks!!